.github/workflows/build.yml

```diff
@@ -37,8 +135,123 @@ jobs:
         if: always()
         run: |
           ve1/bin/pr-artifact --directory=./pr --pr-number=${{ github.event.number }} --api-url=${{ github.event.pull_request._links.self.href }}
+
       - uses: actions/upload-artifact@v2
         if: always()
         with:
           name: pr
           path: pr/
+
+      - name: 'Prepare PR failure comment'
+        if: ${{ steps.verify_pr.conclusion == 'failure' }}
+        run: |
+          python3 scripts/prepare_pr_comment.py failure ${{ github.repository }}
+
+      - name: 'Prepare PR success comment'
+        if: ${{ steps.verify_pr.conclusion == 'success' }}
+        run: |
+          python3 scripts/prepare_pr_comment.py success ${{ github.repository }}
+
+      - name: 'Comment on PR'
+        if: always()
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            var fs = require('fs');
+            var issue_number = fs.readFileSync('./pr/NR');
+            var comment = fs.readFileSync('./pr/comment', {encoding:'utf8', flag:'r'});
+            github.issues.createComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: Number(issue_number),
+              body: comment
+            });
+
+      - name: 'Add label to PR'
+        if: always()
```

Comment:
is this line really needed?

Reply:
Removed. 76f652c5dcc1eb7d3443167b1c25c64c3fade50c